File: app/src/main/java/com/example/javakotlin/javakotlinoverview/java/IfPlusForLoop.java
package com.example.javakotlin.javakotlinoverview.java;

public class IfPlusForLoop {
    public static void main(String args[]) {
        int i;
        for(i=0;i<10;i++) {
            if(i%2 == 0) {
                System.out.println(i + " divisible by 2");
            }
            else {
                continue;
            }
        }
    }
}

File: app/src/main/java/com/example/javakotlin/javakotlinoverview/kotlin/IfPlusForLoop.kt
package com.example.javakotlin.javakotlinoverview.kotlin

class IfPlusForLoop {
    fun main(args: Array<String>) {
        for(i in 1..10) {
            if(i%2 == 0) {
                println("$i divisible by 2")
            }
            else {
                continue
            }
        }
    }
}